include/seqan/basic/iterator_position.h

```diff
@@ -168,10 +177,10 @@ container(Iter<TContainer, PositionIterator> & me)
 }
 
 template <typename TContainer>
-inline typename Parameter_<TContainer>::Type
+inline typename Parameter_<TContainer const>::Type
 container(Iter<TContainer, PositionIterator> const & me)
 {
-    return _toParameter<TContainer>(me.data_container);
+    return _toParameter<TContainer const>(me.data_container);
```

Comment:
Is that correct? To make the container const? I thought, a const-iterator can return a non-const container. The constness of the container type should be independent of the constness of the iterator, right?